src/workflows-app/components/FilterableWorkflowTable.ts

```diff
@@ -151,6 +206,57 @@ const FilterableWorkflowTable = ({
     }
   };
 
+  const includeKeys = useMemo(
+    () => [
+      'backendStatus',
+      'executionStatus',
+      'shardIndex',
+      'jobId',
+      'start',
+      'end',
+      'stderr',
+      'stdout',
+      'tes_stdout',
+      'tes_stderr',
+      'attempt',
+      'subWorkflowId', // needed for task type column
+    ],
+    []
+  );
+  const excludeKeys = useMemo(() => [], []);
+
+  const stripTaskPrefixFromKeys = (keyValuePairs: {}): {} => {
```

Comment:
It would be nice to have more specific types for this function. I haven't tested this suggestion, but I think it would work.

```suggestion
  const stripTaskPrefixFromKeys = <T extends Record<string, any>>(keyValuePairs: T): T => {
```

Reply:
Also, it looks like this function doesn't depend on any component props or state, so consider moving it outside the function where it can be tested independently.